pyoptools/gui/ipynbplotutils.py

```diff
@@ -177,7 +177,7 @@ def _repr_jpeg_(self):
 
     def _buffer_to_image(self):
         h, w, _c = self.buffer.shape
-        image = PImage.fromstring("RGBA", (w, h), self.buffer)
+        image = PImage.frombytes("RGBA", (w, h), self.buffer)
```

Comment:
Does the `frombytes` method works well with python 2.7? just matter of keeping compatibility